main.tf

```diff
@@ -41,6 +40,10 @@ locals {
         secret_name    = secret_name
         secret_data    = secret_data
   }]])
+
+  firewall_name  = format("%s-%s", "ingress-allow", var.name_suffix)
+  istio_ports    = ["10250", "443", "15017", "8080", "15000"]
```

Comment:
Let's break this line down like what roman has done in another PR. Basically showing which ports have what significance.

Reply:
done